server/auth/sso/sso_test.go

```diff
@@ -153,3 +158,68 @@ func TestGetSessionExpiry(t *testing.T) {
 	}
 	assert.Equal(t, config.GetSessionExpiry(), 5*time.Hour)
 }
+
+func TestLoadSsoClientIdAndSecretFromEnv(t *testing.T) {
+	fakeClient := fake.NewSimpleClientset(ssoConfigSecret).CoreV1().Secrets(testNamespace)
+
+	os.Setenv("ARGO_CLIENT_ID", "sso-client-id-value")
+	os.Setenv("ARGO_CLIENT_SECRET", "sso-client-secret-value")
```

Comment:
you'll want to unset these at the end of the test, which is done in most other tests

Reply:
```suggestion
	t.Setenv("ARGO_CLIENT_ID", "sso-client-id-value")
	t.Setenv("ARGO_CLIENT_SECRET", "sso-client-secret-value")
```

there's a helper for that